packages/foam-core/src/note-graph.ts

```diff
@@ -91,7 +91,7 @@ export class NoteGraph implements NoteGraphAPI {
     // prettier-ignore
     const filterFn =
       query == null ? (note: Note | null) => note != null
-        : 'slug' in query ? (note: Note | null) => note?.slug === query.slug
+        : 'slug' in query ? (note: Note | null) => [nameToSlug(query.slug), query.slug].includes(note?.slug as string)
```

Comment:
I am checking for both [[Slug Name]] and [[slug-name]] here, however they are defined in the note, so both can be used interchangably.

Reply:
@riccardoferretti

Reply:
@SanketDG I have a feeling this wouldn't work with the dataviz graph, have you tested it?

Reply:
Yes it definitely works with my notes, which all have filenames like "javascript-execution.md" and referenced as [[Javascript Execution]], this is uniform throughout.

Any other cases you want me to check?